### ISSUE TITLE
Error querying suites info when installed without public synonyms

### ISSUE THREAD

**Issue by michaelstaszewski** [utPLSQL/utPLSQL#957]
Using 3.1.7, but I don't believe this is a new issue.

I've followed the [install guide](https://github.com/utPLSQL/utPLSQL/blob/develop/docs/userguide/install.md) to install privately into a schema, UT3, and I've allowed access to another schema, UT_USER. My installation process is...

```sql
@create_utplsql_owner.sql ut3 ut3 users
@install.sql ut3

CREATE USER ut_user IDENTIFIED BY ut_user;
GRANT "CONNECT" TO ut_user;
GRANT "RESOURCE" TO ut_user;
ALTER USER ut_user DEFAULT ROLE "CONNECT", "RESOURCE";

@create_user_grants.sql ut3 ut_user;
@create_user_synonyms.sql ut3 ut_user;
```

When connected as UT_USER I can run suites OK, but I cannot query for them using [get_suites_info](https://github.com/utPLSQL/utPLSQL/blob/develop/docs/userguide/querying_suites.md).

#### Query
```sql
  SELECT t.*
    FROM TABLE (
             ut_runner.get_suites_info (a_owner => user, a_package_name => null))
         t
```
#### Exception
```
ORA-00904: : invalid identifier
ORA-06512: at "UT3.UT_SUITE_MANAGER", line 821
ORA-06512: at "UT3.UT_RUNNER", line 201
ORA-06512: at line 1
```

**Comment by jgebal** [comment]
Thanks for reporting this issue @michaelstaszewski 
Looks like grants are missing for suite cache tables when using non-public grants.
You would need to add those to the installation:
```sql
grant select on &&ut3_owner..ut_suite_cache to &ut3_user;
grant select on &&ut3_owner..ut_suite_cache_package to &ut3_user;
grant select on &&ut3_owner..ut_suite_cache_schema to &ut3_user;
```
I'm currently working on improving utPLSQL security and privileges model so it's possible that those grants will go away totally in next release.

**Comment by jgebal** [comment]
I've replicated the issue locally and provided a permanent fix for it in PR #954 
If you want, you can use the code in that PR for now.
The changes will become part of `v3.1.8`.

Sorry for inconvenience.
